/robot/force_dimension:
  ros__parameters:
    sample_interval_s: 0.0005
    feedback_sample_decimation:
      position: 50
      velocity: 200
      orientation: 50
      button: 50
      gripper_gap: 50
      gripper_angle: 50
      state: 50
    device_state_metrics:
      include_position: true
      include_velocity: true
      include_orientation: true
      include_gripper: true
      include_buttons: true
    disable_hardware: false
